src/Core/Services/MetadataProviders/MsSqlMetadataProvider.cs

```diff
@@ -102,7 +102,11 @@ public override async Task PopulateTriggerMetadataForTable(string entityName, st
 
                     if (hasDefault)
                     {
-                        columnDefinition.DefaultValue = columnInfo["COLUMN_DEFAULT"];
+                        // If default value is a built_in function let database handle it.
+                        if (!IsBuiltInFunction(columnInfo["COLUMN_DEFAULT"].ToString()!))
+                        {
+                            columnDefinition.DefaultValue = columnInfo["COLUMN_DEFAULT"];
```

Comment:
Do we also ensure that if the column has as builtin function as a default, we make it nullable so it is not required in the input?

Reply:
yes, we are making sure of that in the `CreateMutationBuilder.cs` line:129